src/main/kotlin/no/nav/tilleggsstonader/sak/vedtak/totrinnskontroll/TotrinnskontrollService.kt

```diff
@@ -27,35 +33,50 @@ class TotrinnskontrollService(
     private val behandlingshistorikkService: BehandlingshistorikkService,
     private val behandlingService: BehandlingService,
     private val tilgangService: TilgangService,
+    private val totrinnskontrollRepository: TotrinnskontrollRepository,
 ) {
 
     /**
      * Lagrer data om besluttning av totrinnskontroll
      * og returnerer navIdent til saksbehandleren som sendte behandling til beslutter
      */
+
     @Transactional
-    fun lagreTotrinnskontrollOgReturnerBehandler(
+    fun opprettTotrinnskontroll(saksbehandling: Saksbehandling, beslutteVedtak: BeslutteVedtakDto): Totrinnskontroll {
```

Comment:
Er det denne som skal brukes når en behandling sendes til beslutter, eller når beslutter godkjenner/underkjenner vedtaket?

Reply:
Det blir når man sender til totrinnskontroll
`lagreTotrinnskontrollOgReturnerSaksbehandler` som kanskje burde burde bli "lagreUtfall" eller noe liknende, blir brukt for å oppdatere statusen, dvs godkjenner/underkjenner/avbryter

Reply:
Ok, men da skal vel ikke `beslutteVedtak` være med eller?

Reply:
Det er riktig :)

Reply:
beslutteVedtak tatt ut.